libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -389,6 +395,27 @@ public void run() {
         return view;
     }
 
+    private String getAppTheme() {
```

Comment:
I'm not really sure about the details of GB implementation, but this approach of getting string resource from the `app_theme_entry_value_*` looks a bit hacky to me. Can GB be aware of the current theme from withing its own code? Or we could just use boolean to indicate light or dark mode.

Reply:
Fixed. Changed to boolean.